packages/vue2/vue2-vanilla/package.json

```diff
@@ -59,11 +59,10 @@
     "@jsonforms/vue2": "^3.0.0-beta.5",
     "@rollup/plugin-alias": "^3.1.8",
     "@types/jest": "^24.0.23",
-    "@vue/cli-plugin-babel": "~4.5.0",
-    "@vue/cli-plugin-typescript": "~4.5.0",
-    "@vue/cli-plugin-unit-jest": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
-    "@vue/composition-api": "^1.4.2",
+    "@vue/cli-plugin-babel": "~4.5.18",
+    "@vue/cli-plugin-typescript": "~4.5.18",
+    "@vue/cli-plugin-unit-jest": "~4.5.18",
+    "@vue/cli-service": "~4.5.18",
```

Comment:
Do we need to raise these dependencies? If yes please align them in the Vue 3 packages. If possible any dependency within the monorepo should be required in the same version.

Reply:
Yes it should be at least `~4.5.18` for v4 or `~5.0.6 ` for v5 . See [here](https://github.com/vuejs/vue/blob/main/CHANGELOG.md#vue-cli--webpack). I align the Vue 3 packages as well to `~4.5.18`